jenkins-scripts/README.md

```diff
@@ -0,0 +1,39 @@
+# Jenkins scripts
+
+We use [DSL Jenkins plugin](https://plugins.jenkins.io/job-dsl/) to allow us to manage and define the jobs as human readable code. You can find the different job configs under the [`dsl`](./dsl/) folder. 
+
+## Useful links
+- [List of installed plugins in Jenkins](https://github.com/osrf/chef-osrf/blob/latest/cookbooks/osrfbuild/attributes/plugins.rb)
+- [Jenkins DSL API docs](https://jenkinsci.github.io/job-dsl-plugin/)
+- [Jenkins DSL Wiki](https://github.com/jenkinsci/job-dsl-plugin/wiki) 
+
+
+## Local testing
+
+To test locally the build of the different `dsl` jobs you need the following: 
+
+1. Download `job_dsl-core.standalone.jar` to be able to build the dsl jobs locally.  [Download here](https://repo.jenkins-ci.org/artifactory/releases/org/jenkins-ci/plugins/job-dsl-core/1.77/)
```

Comment:
May be a good moment to suggest to the use same exact version that the server is shipping and link the relevant resource: this one https://github.com/osrf/chef-osrf/blob/latest/cookbooks/osrfbuild/attributes/plugins.rb#L70